scripts/downgrade-users/example.csv

```diff
@@ -0,0 +1,3 @@
+userId,displayName,userName,totalCount,affiliationId,permission,organizationId,acronym,name,slug
```

Comment:
It looks like in the script we're only using the orgId and userName values. Are the other headers necessary?

Reply:
Nope, not required for the script. The other stuff is there since that's the info we took from the export and it might be easier to vet with the extra columns. We can remove from example.csv no problem